lib/doctor/docker.rb

```diff
@@ -0,0 +1,35 @@
+module Doctor
+  class Docker
+    def initialize
+      @message = []
+    end
+
+    def call
+      get_install_state
+
+      message.join
+    end
+
+    private
+    attr_reader :message
+
+    DOCKER_INSTALLED = "✅ Docker is installed".freeze
+    INSTALL_DOCKER = <<~HEREDOC.freeze
+      ❌ Docker not found.
+      You should install docker by grabbing the latest image from https://docs.docker.com/docker-for-mac/release-notes/.
+      For manual installation, visit https://docs.docker.com/install/.
+    HEREDOC
+
+    def get_install_state
```

Comment:
Can this method be called `install_state`? or `add_install_state_message`

Reply:
👍I've changed it to `install_state?`. Also changed the corresponding names in https://github.com/alphagov/govuk-docker/blob/18420192368c1a1d1bfc71d0e5639efd16c4b06e/lib/doctor/dnsmasq.rb